backend/app/processing/llm.py

```
@@ -0,0 +1,369 @@
+import json
+import requests
+from typing import Dict, Any, List, Optional
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)